cmd/brew-createzap

```diff
@@ -8,7 +8,7 @@
 #     brew createzap appname
 #
 
-readonly TMP_FILE="/tmp/brewcask-createzap"
+readonly TMP_FILE=$(mktemp /tmp/brewcask-createzap.XXXXXX)
```

Comment:
Use mktemp for collision-free tmp file creation. Allows running multiple instances of createzap at the same time.